base/dict.jl

```diff
@@ -493,6 +561,12 @@ julia> get(d, "a", 3)
 julia> get(d, "c", 3)
 3
 ```
+
+Using `get()` is the same as using `d[x]` syntax when the key exists in the collection. However, when the key doesn't exist, `get` returns the default value while `d[x]` throws an error.
```

Comment:
This still just seems like a rewrite of the section above the example above this. I think it can be removed.

Reply:
The section above doesn't say the difference between `d[x]` and `get()`